googlepay/src/main/java/com/adyen/checkout/googlepay/GooglePayComponent.kt

```diff
@@ -74,10 +83,27 @@ class GooglePayComponent internal constructor(
      * @param activity    The activity to start the screen and later receive the result.
      * @param requestCode The code that will be returned on the [Activity.onActivityResult]
      */
+    @Deprecated("Deprecated in favor of startGooglePayScreen()", ReplaceWith("submit()"))
```

Comment:
Deprecated in favor of `submit()` I guess?

Reply:
Good catch 🦅